feature/floor-map/src/main/java/io/github/droidkaigi/confsched2023/floormap/component/FloorMapSideEventItem.kt

```diff
@@ -52,55 +60,77 @@ fun FloorMapSideEventItem(
+                    )
+                    Spacer(modifier = Modifier.width(8.dp))
+                    Text(
+                        text = sideEvent.title.currentLangTitle,
+                        style = MaterialTheme.typography.labelLarge,
+                        maxLines = 1,
+                        overflow = TextOverflow.Ellipsis,
+                    )
+                }
+                Spacer(modifier = Modifier.height(4.dp))
                 Text(
-                    text = createAnnotatedEventDetailString(),
-                    style = MaterialTheme.typography.bodySmall,
-                    modifier = Modifier.clickable {
-                        // todo open link
-                    },
+                    text = sideEvent.description.currentLangTitle,
+                    style = MaterialTheme.typography.bodyMedium,
+                )
+                Spacer(modifier = Modifier.height(8.dp))
+                Row(verticalAlignment = Alignment.CenterVertically) {
+                    Icon(
+                        imageVector = Icons.Filled.Schedule,
+                        contentDescription = FavoriteIcon.asString(),
+                        modifier = Modifier.size(16.dp),
+                    )
+                    Spacer(modifier = Modifier.width(8.dp))
+                    Text(
+                        text = sideEvent.timeText.currentLangTitle,
+                        style = MaterialTheme.typography.bodySmall,
+                    )
+                }
+                if (sideEvent.link != null) {
+                    Spacer(modifier = Modifier.height(8.dp))
+                    Row(verticalAlignment = Alignment.CenterVertically) {
+                        Icon(
+                            imageVector = Icons.Filled.Link,
+                            contentDescription = FavoriteIcon.asString(),
+                            modifier = Modifier.size(16.dp),
+                        )
+                        Spacer(modifier = Modifier.width(8.dp))
+                        Text(
+                            text = createAnnotatedEventDetailString(),
+                            style = MaterialTheme.typography.bodySmall,
+                            modifier = Modifier.clickable {
+                                // todo open link
+                            },
+                        )
+                    }
+                }
+            }
+            if (sideEvent.imageLink != null) {
+                Image(
+                    modifier = Modifier
+                        .size(80.dp)
+                        .clip(RoundedCornerShape(16.dp))
+                        .border(
+                            width = 1.dp,
+                            color = MaterialTheme.colorScheme.outline,
+                            shape = RoundedCornerShape(16.dp),
+                        ),
+                    painter = rememberAsyncImagePainter(sideEvent.imageLink!!),
```

Comment:
Could you use `previewOverride`? Because this causes flaky screenshot tests since it involves a network request.

Reply:
sorry, missed it. c26a7f5